content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -101,13 +108,37 @@ See the Google Kubernetes Engine (GKE) guide to
 
 1. Check the [Kubeflow Pipelines GitHub repository](https://github.com/kubeflow/pipelines/releases) for available releases.
 
-1. Upgrade to a version of Kubeflow Pipelines standalone:
+1. Upgrade to a version of Kubeflow Pipelines standalone is the same as deploying that version:
```

Comment:
I would recommend removing "is the same as deploying that version" and adding that this is for 0.4 and higher. Something like:

For Kubeflow Pipelines 0.4.0 and higher, use the following commands to upgrade Kubeflow Pipelines standalone to a higher version.

Reply:
I'd like to clarify that upgrading to a version uses the same command as installing that version. To avoid having to duplicate instructions for each version.

Adjusting on above, what do you think above
```
For upgrading to Kubeflow Pipelines 0.4.0 and higher, use the following commands (it's the same as installing corresponding version):
```

Reply:
For people who want to upgrade to the current version, this information isn't necessary. Let's add this to the instructions for upgrading prior versions.

To upgrade to Kubeflow Pipelines 0.3.0 and lower, use the [deployment instructions](#deploying-kubeflow-pipelines) to upgrade your Kubeflow Pipelines cluster.

Reply:
Thanks! That sounds great